src/course/syllabus/apprenticeship/database/project.md

```diff
@@ -44,3 +44,60 @@ Since this project is open-ended you'll need to write your own more specific use
 - Founders & Coders book sharing system
 - Food / coffee recommendations around Founders & Coders
 - Founders & Coders events calendar
+
+## Deploying persistent volumes to Fly.io
+
+A simple mental model for app deployment is that we’re installing and running our app on a computer in the cloud. Yet this is an oversimplification and an important difference is that the PaaS services we use for deployment (ie Fly) have an ephemeral file system.
+
+An ephemeral file system is a type of file system that is not permanently stored on a device or disk. It exists only for a short period of time, and any files or data stored in it are not persisted after the session ends or the file system is deleted. This type of file system is usually used in cloud computing environment as it allows for greater flexibility and efficiency compared to using permanent storage.
+
+As Fly uses an ephemeral file system and SQLite works by storing the database as a file on the system we need to make special provisions for deployment. If we didn’t, then our database would disappear every time the file system is restarted.
+
+To solve this we’re going to use Fly’s [Volume’s feature](https://fly.io/docs/reference/volumes/).
+
+### Adding a volume to your app
+
+1. [Install the Fly CLI](https://oliverjam.es/articles/deploying-to-fly#how-do-you-deploy) if you haven’t already.
```

Comment:
Perhaps clarify with:

[Install the Fly CLI and deploy you app](https://oliverjam.es/articles/deploying-to-fly#how-do-you-deploy) if you haven’t already.